debian/changelog

```diff
@@ -1,29 +1,32 @@
-cloud-init (23.1-0ubuntu0~22.04.2) UNRELEASED; urgency=medium
+cloud-init (23.1.1-0ubuntu0~22.04.1) UNRELEASED; urgency=medium
```

Comment:
UNRELEASED version numbers don't really matter for our tooling and daily build recipes outside of the primary version prefix in front of the hyphen `-` which out `build-package` script uses.
  But, we probably should actually increment this UNRELEASED pkg version to 23.1.2 because we know our quilt patches no longer apply cleanly after our quilt refresh a month ago. This will allow us to continue running test builds with `build-package` locally without hitting build-package failure with errors like:
```
dpkg-source: info: the patch has fuzz which is not allowed, or is malformed
dpkg-source: info: if patch 'expire-on-hashed-users.patch' is correctly applied by quilt, use 'quilt refresh' to update it
```

```suggestion
cloud-init (23.1.2-0ubuntu0~22.04.1) UNRELEASED; urgency=medium
```